rasa/core/policies/policy.py

```diff
@@ -602,3 +628,71 @@ def confidence_scores_for(
     results[idx] = value
 
     return results
+
+
+class InvalidPolicyConfig(RasaException):
+    """Exception that can be raised when policy config is not valid."""
+
+
+# TODO: Test the shit out of this
```

Comment:
haha - will this be part of this PR?

Reply:
yeah, unfortunately. Should've been tested before in my opinion 😬

Reply:
done